kernel/linux/modules/netsupport.mk

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+NETWORK_SUPPORT_MENU:=Network Support
+
+define KernelPackage/openvswitch
+  SECTION:=kernel
+  CATEGORY:=Kernel modules
+  SUBMENU:=Network Support
+  TITLE:=Open vSwitch Kernel Package
+  KCONFIG:= \
+	CONFIG_BRIDGE \
+	CONFIG_OPENVSWITCH \
+	CONFIG_OPENVSWITCH_GRE=n \
+	CONFIG_OPENVSWITCH_VXLAN=n \
+	CONFIG_OPENVSWITCH_GENEVE=n
+  DEPENDS:= \
+	@IPV6 +kmod-gre +kmod-lib-crc32c +kmod-mpls \
```

Comment:
Why would mpls be a dependency?  Or gre?

Reply:
OVS requires the mpls kmod for some reason ; 
no idea why : 
https://github.com/torvalds/linux/blob/master/net/openvswitch/Kconfig#L14

for GRE, I'm not sure anymore ;
there used to be a time when kmod OVS would not load without the GRE kmod
that may not be the case anymore

Reply:
Could you try pulling things out until it breaks?

Reply:
I think I already did that, but will retry